recipes/valerius/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "valerius" %}
+{% set version = "0.2" %}
```

Comment:
Isn't this package for suitable for bioconda instead, https://bioconda.github.io?

Reply:
We generall don't mind having it here though either